sktime/classification/compose/_ensemble.py

```diff
@@ -314,16 +308,19 @@ def _validate_estimator(self):
     def predict(self, X):
         """
         Predict class for X.
```

Comment:
Move this comment up a line so:
`
"""Predict class for X.

Reply:
Done 👍